.gitlab-ci.yml

```
@@ -194,6 +194,9 @@ onboarding_dotnet:
         - ONBOARDING_FILTER_WEBLOG: [test-app-dotnet-multicontainer,test-app-dotnet-multialpine]
           SCENARIO: [SIMPLE_INSTALLER_AUTO_INJECTION]
           DEFAULT_VMS: ["True", "False"]
+        - ONBOARDING_FILTER_WEBLOG: [test-app-dotnet-unsupported]
+          SCENARIO: [INSTALLER_NOT_SUPPORTED_AUTO_INJECTION]
+          DEFAULT_VMS: ["True", "False"]
```

```suggestion
          DEFAULT_VMS: ["True"]
```

Since we skip all the systems in the non-default we can probably remove the `False` here.

However, maybe we want to keep it anyways in case we add support later, or move the tested VM to non-default?

No preference, but the job run with `False` just does a single skip.

---

Better to maintain the both cases (true and false)